src/celestia/sdl/CMakeLists.txt

```
@@ -82,10 +82,22 @@ else()
   target_link_libraries(imgui PUBLIC ${SDL2_LIBRARIES})
 endif()
 
-install(
-  TARGETS celestia-sdl
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-  COMPONENT sdlgui
-)
+if(WIN32)
+  install(
+    TARGETS celestia-sdl
+    RUNTIME_DEPENDENCIES
+      PRE_EXCLUDE_REGEXES "^api-ms" "^ext-ms-"
+      POST_EXCLUDE_REGEXES ".*system32/.*\\.dll$"
+      DIRECTORIES $<TARGET_FILE_DIR:celestia> $<TARGET_FILE_DIR:SDL2::SDL2>
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    COMPONENT sdlgui
+  )
```

```suggestion
  if(TARGET SDL2::SDL2)
    install(
      TARGETS celestia-sdl
      RUNTIME_DEPENDENCIES
        PRE_EXCLUDE_REGEXES "^api-ms" "^ext-ms-"
        POST_EXCLUDE_REGEXES ".*system32/.*\\.dll$"
        DIRECTORIES $<TARGET_FILE_DIR:celestia> $<TARGET_FILE_DIR:SDL2::SDL2>
      RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
      COMPONENT sdlgui
    )
  else()
    install(
      TARGETS celestia-sdl
      RUNTIME_DEPENDENCIES
        PRE_EXCLUDE_REGEXES "^api-ms" "^ext-ms-"
        POST_EXCLUDE_REGEXES ".*system32/.*\\.dll$"
        DIRECTORIES $<TARGET_FILE_DIR:celestia> $<TARGET_FILE_DIR:${SDL2_LIBRARIES}>
      RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
      COMPONENT sdlgui
    )
  endif()
```

Is this needed? The above if check was introduced in https://github.com/CelestiaProject/Celestia/commit/5c17a06661965f2ed4095897e08ba1710b3e2d80 so maybe @podsvirov would know?

---

> *Added in version 3.19.*
>
> This module defines the following [`IMPORTED`](https://cmake.org/cmake/help/v3.31/prop_tgt/IMPORTED.html#prop_tgt:IMPORTED) target:
>
> `SDL::SDL`
> - The SDL library, if found

--- from https://cmake.org/cmake/help/v3.31/module/FindSDL.html

[related code](https://github.com/Kitware/CMake/blob/2839c3a93312454d83ec898b0709ba2dd220dbfb/Modules/FindSDL.cmake#L296-L301) (added in https://github.com/Kitware/CMake/commit/9ffd2c70bff9eddc3cd056889212a3d12cdd9ead)

And considering [we set minimum required CMake version on Windows to `3.19`](https://github.com/CelestiaProject/Celestia/blob/b3a7b5ec7bcec48294a6e9d69bd8f8a6337d716c/CMakeLists.txt#L2-L3), `find_package` should always provide us a target (if I'm understanding correctly)?

I also tested with SDL2 2.0.14 ([which doesn't set a target AFAICT](https://github.com/libsdl-org/SDL/blob/release-2.0.14/CMakeLists.txt)) and I had no issues building/installing.

Do note that I don't have much experience with CMake so I might be overlooking something.

---

The FindSDL2 module is actually provided by SDL, not CMake. From what I can see, the targets were available from SDL 2.0.12. Debian Bullseye (oldstable) has 2.0.14, so I think we're safe to delete the non-targets branch.